<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interfaz</class>
 <widget class="QMainWindow" name="Interfaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bomberman</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagenes/icono.ico</normaloff>imagenes/icono.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>701</width>
      <height>671</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>170</red>
       <green>170</green>
       <blue>170</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Bebas Neue&quot;;</string>
    </property>
    <property name="text">
     <string>Tiempo</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_score">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lifes">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Bebas Neue&quot;;</string>
    </property>
    <property name="text">
     <string>VIDAS</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_lifes">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>